src/Models/LagrangianParticleTracking/LagrangianParticleTracking.jl

```diff
@@ -120,7 +121,7 @@ include("lagrangian_particle_advection.jl")
 
 update_lagrangian_particles!(model, Δt) = update_lagrangian_particles!(model.particles, model, Δt)
 update_lagrangian_particles!(::Nothing, model, Δt) = nothing
-update_lagrangian_particle_properties!(::ShallowWaterModel, args...) = nothing
+update_lagrangian_particle_properties!(::ShallowWaterModel, Δt) = nothing
```

Comment:
```suggestion
update_lagrangian_particle_properties!(model, Δt) = nothing
```

Reply:
Extend `update_lagrangian_particle_properties!` for the models that support particles

Reply:
Where does this get called too? I'm confused why its needed